nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Analyze/CompatabilityChecker.cs

```diff
@@ -207,6 +207,21 @@ internal static PackageReaders ReadPackage(string tempPackagePath)
         return null;
     }
 
-    internal static string GetTempPackagePath(PackageIdentity package, NuGetContext context)
-        => Path.Combine(context.TempPackageDirectory, package.Id + "." + package.Version + ".nupkg");
+    internal static string GetPackagePath(PackageIdentity package, NuGetContext context)
```

Comment:
This function is the real fix.  All other changes are to make the unit tests more consistent.